.github/workflows/main.yml

```diff
@@ -68,3 +68,30 @@ jobs:
         env: # Set the secret as an input
           DOCKER_USER: ${{ secrets.DOCKER_USER }}
           DOCKER_PASSWD: ${{ secrets.DOCKER_PASSWD }}
+
+
+  test-gpu:
+    environment: ci_test
+    strategy:
+      fail-fast: false
+      matrix:
+        exasol_version:
+          - "default"
+        python_version:
+          - "3.10"
+    runs-on:
+      labels: int-linux-x64-4core-gpu-t4-ubuntu24.04-1
+    name: GPU Test
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@1.0.1
+        with:
+          python-version: '${{ matrix.python_version }}'
+          poetry-version: '2.1.2'
+
+      - name: Allow unprivileged user namespaces
+        run: sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
+      - name: Run GPU tests
+        run: ./scripts/test/run_test.sh "test/test_docker_api_run_db_gpu_test.py"
```

Comment:
Not nice, but I propose to improve this when we migrate the integration tests to pytest